msvc/config.h

```diff
@@ -0,0 +1,4 @@
+#define HAVE_AVX2   1
+#define HAVE_NEON32 0
+#define HAVE_NEON64 0
+#define HAVE_SSSE3  1
```

Comment:
Does this file belong in version control? It should be autogenerated during compilation. Or is this under the assumption that Windows always runs x86, and that we will rely only on runtime feature checking?